server/testutils/factories/referral.ts

```diff
@@ -9,4 +9,10 @@ export default Factory.define<Referral>(() => ({
   offeringId: faker.string.uuid(),
   prisonNumber: faker.string.alphanumeric({ length: 7 }),
   referrerId: faker.string.numeric({ length: 6 }),
+  status: faker.helpers.arrayElement([
```

Comment:
Could default to one of the values instead if we think that's better.

Reply:
I think the range of options is ok here, we'll likely be explicitly setting it in tests if we're testing anything related to this field so the default value wouldn't get used anyway. I think it's good to have a range.